elasticsearch/run.sh

```diff
@@ -216,4 +216,8 @@ verify_or_add_index_templates() {
 
 verify_or_add_index_templates &
 
+if [ -v HEAP_DUMP_LOCATION ]; then
+    info Setting heap dump location "$HEAP_DUMP_LOCATION"
+    export JAVA_OPTS="${JAVA_OPTS:-} -XX:HeapDumpPath=$HEAP_DUMP_LOCATION"
```

Comment:
if `HeapDumpPath` is unset, java will write the dump file to the root partition (which was causing some issues)

[this](https://github.com/openshift/openshift-ansible/pull/4451) PR sets the `HEAP_DUMP_LOCATION` variable in openshift-ansible to `/elasticsearch/persistent`